app/bundles/ProjectBundle/DTO/EntityTypeConfig.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\ProjectBundle\DTO;
+
+readonly class EntityTypeConfig
```

Comment:
I love this minimal DTO class! Let's make it [final](https://ocramius.github.io/blog/when-to-declare-classes-final/). There is nothing to inherit anyway :)
```suggestion
final readonly class EntityTypeConfig
```